linux/buildScript.sh

```diff
@@ -11,12 +11,12 @@ set -u
 
 srctgz="https://www.census.gov/ts/x13as/unix/x13ashtmlsrc_V1.1_B19.tar.gz"
 file=$(basename ${srctgz})
-
 td=$(tempfile -p x13)
+rm -r $td
 mkdir ${td}
 cd ${td}
 wget ${srctgz}
-tar xfz ${file}
+tar xf ${file}
```

Comment:
Strange. I do not recall that happening to me. 

Also, you started with the _wrong_ script.  The ones in v1.1.26 are newer ones.